bot.go

```diff
@@ -24,6 +24,7 @@ type Bot struct {
 	writer       *irc.Encoder
 	conn         net.Conn
 	tries        float64
+	stop         chan bool
```

Comment:
In Go we usually use a `chan struct{}` when we're using a channel and don't care about the contents of the channel. This is because the size of an empty `struct{}{}` is 0, so we don't waste any memory.